src/main/java/com/thealgorithms/others/LineSweep.java

```diff
@@ -0,0 +1,41 @@
+package com.thealgorithms.others;
+import java.util.Arrays;
+
+public class LineSweep {
+
+     /** Line Sweep algorithm can be used to solve range problems by first sorting the list of ranges
+     * by the start value of the range in non-decreasing order and doing a "sweep" through the number
+     * line(x-axis) by incrementing the start point by 1 and decrementing end point by -1 on the
+     * number line. https://www.topcoder.com/thrive/articles/Line%20Sweep%20Algorithms
+     * /
+
+     * Find if any ranges overlap
+     * param = ranges Array of range[start,end]
+     * param = maximum Maximum length of number line or largest "end" value
+     * return true if there overlap exists false
+     * otherwise.
+     */
+
+    public static boolean isOverlap(int[][] ranges,int maximum) {
```

Comment:
What is maximum? If it's the biggest point in ranges, it can be derived from ranges and doesn't need to be passed separately

Reply:
You are right. I will add logic to get the maximum value(Basically the largest end value in the array)